src/display/canvas.js

```diff
@@ -330,40 +338,19 @@ class CanvasExtraState {
     this.activeSMask = null;
     this.transferMaps = "none";
 
-    this.startNewPathAndClipBox([0, 0, width, height]);
+    this.minMax = MIN_MAX_INIT.slice();
+    this.clipBox = new Float32Array([0, 0, width, height]);
```

Comment:
Nit: Elsewhere below these fields are sorted in alphabetical order, so please let's do that here as well.
```suggestion
    this.clipBox = new Float32Array([0, 0, width, height]);
    this.minMax = MIN_MAX_INIT.slice();
```